src/main/webapp/dashboard/controller/Dashboard.controller.js

```diff
@@ -2,500 +2,193 @@ sap.ui.define(
+                        ]).then(function (responses) {
+                    	// Get a JSON object from each of the responses
+                    	return Promise.all(responses.map(function (response) {
+                    	    if(response.ok){
+                    	        defined = true;
+                    	        return response.json();
+                    	    }
+
+                    	}));
+                        }).then(function (data) {
+                            numResponses++;
+                    	    if(defined){
+                    	        let terminals = data[0].cardTerminals.cardTerminal;
+                    	        anz_Terminals = anz_Terminals + terminals.length;
+                    	        for(let i = 0; i< terminals.length; i++){
+                    	            if(terminals[i].connected)
+                    	            {
+                    	                activeTerminals++;
+                    	            }
+                    	            else{
+                    	                inactiveTerminals++;
+                    	            }
+                    	        }
+                                let cards = data[1].cards.card;
+                                anz_Cards = anz_Cards + cards.length;
+                                activeConnectors++;
+                    	        defined = false;
+                    	    }
+                    	    else{
+                    	        inactiveConnectors++;
+                    	    }
+                    	    if(numResponses == numConfigs){
+                    	        let StatusConnectors = "";
+                    	        let InfoConnectors = "";
+                    	        if(inactiveConnectors > 0){
+                    	            StatusConnectors = "Error";
+                    	            InfoConnectors= "Offline: " + inactiveConnectors;
+                    	        }
+                    	        else{
+                    	            StatusConnectors = "Success";
+                    	            InfoConnectors = "Alles Online";
+                    	        }
+                    	        let StatusTerminals = "";
+                    	        let InfoTerminals = "";
+                    	        if(inactiveTerminals > 0){
+                    	            StatusTerminals = "Error";
+                    	            InfoTerminals = "Offline: " + inactiveTerminals;
+                    	        }
+                    	        else{
+                    	            StatusTerminals = "Success";
+                    	            InfoTerminals = "Alles Online"
+                    	        }
+                    	        content.push({
+                                    "Name" : "Konnektoren",
+                                    "Value": configs.length,
+                                    "Icon": "http://localhost:8080/dashboard/images/Connector.png",
```

Comment:
This should not use an absolute URL. it won't work on https://connector-admin.med-united.health/frontend/. Should be:
"images/Connector.png"

Reply:
If I set the path not absolute and use "images/Card.png" I receive the following error:

GET https://sdk.openui5.org/resources/images/Card.png 404

It is automatically trying to access sdk.openui5.org/recources, thats why I used the absolute URL, how can I avoid accessing the sdk.openui5 url?

Reply:
I am using const url = document.URL; to get the actual URL and set the source of the icon with 
"Icon": url + "dashboard/images/Connector.png";
So this should work for the other URLs too.